velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -418,5 +434,85 @@ TEST_F(StringTest, substring) {
   EXPECT_EQ(substring("da\u6570\u636Eta", -3), "\u636Eta");
 }
 
+TEST_F(StringTest, rpad) {
+  std::string invalidString = "Ψ\xFF\xFFΣΓΔA";
+  std::string invalidPadString = "\xFFΨ\xFF";
+
+  // Null arguments
```

Comment:
No need to test for null inputs for default-null-behavior functions. This is testing the framework, not the function itself.